OsmAnd/src/net/osmand/plus/track/helpers/GpxReaderTask.java

```diff
@@ -69,6 +69,7 @@ public boolean isReading() {
 
 	@Override
 	protected Void doInBackground(Void... voids) {
+		Thread.currentThread().setName("GpxReaderTask");
```

Comment:
GpxReaderTask runs on AsyncTask.THREAD_POOL_EXECUTOR, so renamed thread from the pool could be used in future for other task with wrong name

Reply:
is it wrong aproach in general or it normal to restore thread name on finish?